conbench/app/_plots.py

```diff
@@ -353,8 +503,15 @@ def time_series_plot(history, benchmark, run, height=380, width=1100):
         source=source_mean_over_time,
         legend_label=label,
         name="history",
-        size=4,
+        size=6,
         color="#ccc",
+        line_width=1,
+        selection_color="#76bf5a",  # like bootstrap panel dff0d8, but darker
+        selection_line_color="#5da540",  # same green, again darker
+        # Cannot change the size upon selection
+        # selection_size=10,
+        nonselection_fill_alpha=1.0,
+        nonselection_line_alpha=1.0,
```

Comment:
This is a pragmatic way to define different styles for "selected" vs "non-selected". Documented at https://docs.bokeh.org/en/3.0.3/docs/user_guide/styling/plots.html#selected-and-unselected-glyphs.